packages/web-components/README.md

```diff
@@ -55,3 +55,22 @@ Storybook will watch modules for changes and hot-reload the module when necessar
 `Failed to execute 'define' on 'CustomElementRegistry': the name "my-custom-element-name" has already been used with this registry`
 
 This is a known issue and will indicate that you need to refresh the page. We're working on surfacing a more instructive error message for this case.
+
+## Notes
+
+### Using Yarn 2 with PnP
+
+`@microsoft/fast-components` works out of the box with Yarn 2 and PnP; however, because of how Yarn expects peer dependencies to be satisfied, we need additional configuation for `fluentui/web-components`.
```

Comment:
I’m not sure we need to add a reference as this package is technically unrelated. If we can rework this to omit the reference to the fast-components everything else looks relatively straight forward.

Reply:
Good point, updated!